maintainers/maintainer-list.nix

```diff
@@ -7511,6 +7511,12 @@
     githubId = 28323;
     name = "Peter Simons";
   };
+  petrosagg = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add petrosagg`` and sort it alphabetically.